apidoc/Titanium/UI/ListView.yml

```diff
@@ -270,6 +270,21 @@ events:
         type: Boolean
         platforms: [iphone, ipad]
 
+  - name: rowsselected
```

Comment:
List-views prefer the term "item" over "row". So either `itemsselected` or `multiselection`  would match better

Reply:
Agreed! Changed to 'itemsselected'.